File: src/0058-Length-of-Last-Word/length_of_last_word.py
from typing import List

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s: str = s.strip()
        words: List[str] = s.split()
        ans: int = len(words[-1])
        
        return ans
    
if __name__ == "__main__":
    s: str = "   fly me   to   the moon  "
    solution = Solution()
    
    print(solution.lengthOfLastWord(s))

File: src/0058-Length-of-Last-Word/length_of_last_word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
  s = s.trim()
        
  let words = s.split(" ")

  if (words.length === 0) return 0
  
  return words[words.length - 1].length
};

let s = "   fly me   to   the moon  "
let solution = new Solution()

console.log(solution.lengthOfLastWord(s));
